### ISSUE TITLE
Allow party members to modify party inventory and description without ownership

### ISSUE THREAD

**Issue by schultzcole** [foundryvtt/dnd5e#3021]
As a GM I would like my players to be able to manage their party's inventory and description, but not change other aspects of the party such as experience points, system type, etc.

**Comment by arbron** [comment]
I don't think that's really feasible given how Foundry's ownership model works. We could potentially lock certain parts of the sheet behind a GM check preventing players from changing those even if they are an owner, but not really allow them to manage inventory if they aren't an owner.

**Comment by schultzcole** [comment]
Yeah, it's a dilemma. I think the easiest approach would be to make the players owners, but restrict non gms from editing certain fields.
